### PR TITLE
Light polishing of the introduction section

### PR DESCRIPTION
**Pull request by josh146**
**Description of the Change:**

* Minor polishing of grammar and spelling

* Updates the image to include new logos and plugins

**Benefits:** n/a

**Possible Drawbacks:** n/a

**Related GitHub Issues:** #362

### CODE REVIEW

File: doc/introduction/circuits.rst [hunk 1]
```diff
@@ -14,18 +14,14 @@ Quantum Circuits
     :target: javascript:void(0);
 
 
-In PennyLane, variational quantum circuits are represented as *quantum node* objects. A quantum node
-is a combination of a quantum function that composes the circuit,
-and a device that runs the computation. One can conveniently create quantum nodes using
-the quantum node decorator.
+In PennyLane, quantum circuits are represented as *quantum node* objects. A quantum node is used to 
+declare the quantum circuit, and also ties the computation to a specific device that executes it. 
+Quantum nodes can be easily created by using the :ref:`qnode <intro_vcirc_decorator>` decorator.
```
Comment (josh146) [inline doc/introduction/circuits.rst]: Just out of curiosity, is this change more or less to remove mention of 'quantum function' for clarity?

File: doc/introduction/circuits.rst [hunk 2]
```diff
@@ -70,8 +66,8 @@ constraints:
 
 .. note::
 
-    The quantum operations cannot be used outside of a quantum circuit function, as all
-    :class:`Operations <pennylane.operation.Operation>` must be executed from within a QNode.
+    :doc:`operations` can be created outside of a quantum circuit function; however,
+    quantum operations can only be *executed* from within a QNode.
```
Comment (josh146) [inline doc/introduction/circuits.rst]: Thinking about this, do we want to mention at all that operations can be created outside of a quantum circuit function? The use case for that is quite small, and may lead to odd patterns being attemped.

Comment (josh146) [reply doc/introduction/circuits.rst]: How about just:
```rest
.. note::

    Quantum operations can only be executed on a device from within a QNode.
```

File: doc/introduction/circuits.rst [hunk 3]
```diff
@@ -130,12 +126,12 @@ A QNode can be explicitly created as follows:
...
 .. code-block:: python
 
-    qnode = qml.QNode(my_quantum_function, dev)
+    circuit = qml.QNode(my_quantum_function, dev)
 
 The QNode can be used to compute the result of a quantum circuit as if it was a standard Python
 function. It takes the same arguments as the original quantum function:
 
->>> qnode(np.pi/4, 0.7)
+>>> circuit(np.pi/4, 0.7)
```
Comment (josh146) [inline doc/introduction/circuits.rst]: I like circuit over qnode as well :+1:

File: doc/introduction/pennylane.rst [hunk 4]
```diff
@@ -51,7 +52,7 @@ In the following sections you can learn more about the key features of PennyLane
 the process of programming quantum circuits with trainable parameters.
 
 2. :doc:`interfaces` introduces how PennyLane is used with different
-machine learning libraries to optimize quantum circuits or hybrid computations.
+classical computing libraries to optimize quantum circuits or hybrid computations.
```
Comment (josh146) [inline doc/introduction/pennylane.rst]: 'different optimization libraries'?

Comment (josh146) [reply doc/introduction/pennylane.rst]: classical computing library sounds a bit odd to me (all existing software is a classical computing library in a way)

File: doc/introduction/circuits.rst (hunk 1 above)
Comment (co9olguy) [reply doc/introduction/circuits.rst]: Yep.

File: doc/introduction/circuits.rst (hunk 2 above)
Comment (co9olguy) [reply doc/introduction/circuits.rst]: Had the same thought.

**Comment by co9olguy** [comment]
@josh146 btw the navbar issue was still very present when I was looking at these pages. I didn't do anything to fix that in this PR

**Pull request merged**